python/src/pyx/logs_db/db_mysql.py

```
@@ -1,134 +1,177 @@
-            cursor.execute(query, params)
-
-        conn.commit()
-        return True
-    except pymysql.MySQLError as e:
-        print(f"MySQL Database error: {e}")
-        return e
+        conn = pymysql.connect(**DB_CONFIG)
+        cursor = conn.cursor(pymysql.cursors.DictCursor)
+        yield conn, cursor
+    except MySQLError as e:
+        logger.error(f"Database connection error: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)